Updated Export-StartLayout docs to account for changes to module in W… (#3214)
MicrosoftDocs/windows-powershell-docs: docset/winserver2022-ps/startlayout/Export-StartLayout.md

You can export layout and use Import-StartLayout as well as other management policies for the Start layout.
Specify the .xml file name extension in the name of the file.

**Note:** In Windows 11, this cmdlet exports the Start Menu layout in JSON (.json) file format. 

## EXAMPLES

### Example 1: Export the layout (Windows 10)
```
PS C:\> Export-StartLayout -Path "C:\Layouts\Marketing.xml"
```

This command exports the layout of the tiles on Start to a file named `Marketing.xml` in the C:\Layouts folder.

### Example 2: Export the layout (Windows 11)
```
PS C:\> Export-StartLayout -Path "$env:USERPROFILE\Desktop\CurrentLayout.json"
```

> This command exports the layout of the tiles on Start to a file named `CurrentLayout.json` in the current user's `Desktop` folder.

Reviewer: ```suggestion
This command exports the layout of the tiles on Start to a file named `CurrentLayout.json` in the current user's Desktop folder.
```

### -LiteralPath
Specifies a literal path to a layout file.

> If on Windows 10, include the `.xml` file name extension. If on Windows 11, include the `.json` file name extension.

Reviewer: ```suggestion
In Windows 10, include the `.xml` file name extension. In Windows 11, include the `.json` file name extension.
```
